mesh/ncmesh.cpp

```diff
@@ -4888,24 +4888,24 @@ int NCMesh::GetElementSizeReduction(int i) const
    return reduction;
 }
 
-void NCMesh::GetElementFacesAttributes(int i,
-                                       Array<int> &faces,
-                                       Array<int> &fattr) const
+void NCMesh::GetElementFacesAttributes(int i_,
```

Comment:
Is `i_` really necessary?

Reply:
I was trying for internal consistency but I agree that these trailing underscores are annoying.

I've changed all three arguments to more expressive names. I think this is a better choice. Thanks for the nudge.